src/Symfony/Component/Translation/Translatable.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+use Symfony\Contracts\Translation\TranslatableInterface;
+
+/**
+ * @author Nate Wiebe <nate@northern.co>
+ */
+final class Translatable implements TranslatableInterface
+{
+    private $message;
+
+    private $parameters;
+
+    private $domain;
+
+    public function __construct(string $message, array $parameters = [], ?string $domain = null)
+    {
+        $this->message = $message;
```

Comment:
perhaps a LogicException if $message == '', as this isnt accounted for anymore in `trans()` now

Reply:
Right, will move the object conversion higher so it shouldn't be any different than before.